src/main/java/org/xbill/DNS/ResolverConfig.java

```diff
@@ -52,21 +52,25 @@
 ResolverConfig() {
 	if (findProperty())
 		return;
-	if (findSunJVM())
-		return;
+	boolean attemptedSun = false;
 	if (servers == null || searchlist == null) {
```

Comment:
This check is now useless - findProperty is ony true when either servers or searchList is null